test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
I think it is because they want to test the ajax scenario, (which is not a mainline usecase, but there are some people who wanted to do it.). For ajax scenario a token has to be explicitly passed.